scripts/build.sh

```diff
@@ -4,35 +4,34 @@
 #
 set -ex
 
-
-
+if [ "$CKAN_VERSION" = "2.9" ]; then
+    pip install "setuptools>=44.1.0,<71"
+fi
 if [ "$CKAN_GIT_ORG" != "ckan" ]; then
   SRC_DIR=/srv/app/src
   APP_DIR=/srv/app
 #  PIP_SRC=${SRC_DIR}
   pushd ${APP_DIR}
     echo "pip install -e git+https://github.com/${CKAN_GIT_ORG}/ckan.git@${CKAN_GIT_VERSION}#egg=ckan"
     echo "update manually as its already a git pip installed module"
-    sudo git config --global --add safe.directory "$SRC_DIR/ckan"
+    git config --global --add safe.directory "$SRC_DIR/ckan"
   popd
 
   pushd ${SRC_DIR}/ckan
 
-  sudo git remote set-url origin "https://github.com/${CKAN_GIT_ORG}/ckan.git"
```

Comment:
should we look at trying the pypi version of ckan? https://pypi.org/project/ckan/

Reply:
We want to be able to test this on our fork, as well as on vanilla CKAN (especially since our fork adds relevant hooks), so I think setting the Git remote is the most consistent approach.